src/abstractdataframe/selection.jl

```diff
@@ -428,35 +432,41 @@ SELECT_ARG_RULES =
 
     * Any index that is allowed for column indexing
       ($COLUMNINDEX_STR; $MULTICOLUMNINDEX_STR).
+    * A function or a type
     * Column transformation operations using the `Pair` notation that is
-      described below and vectors of such pairs.
+      described below and vectors or matrices of such pairs.
 
     Columns can be renamed using the `old_column => new_column_name` syntax, and
     transformed using the `old_column => fun => new_column_name` syntax.
-    `new_column_name` must be a `Symbol` or a string, and `fun` a function or a
-    type. If `old_column` is a `Symbol`, a string, or an integer then `fun` is
-    applied to the corresponding column vector. Otherwise `old_column` can be
-    any column indexing syntax, in which case `fun` will be passed the column
-    vectors specified by `old_column` as separate arguments. The only exception
-    is when `old_column` is an `AsTable` type wrapping a selector, in which case
-    `fun` is passed a `NamedTuple` containing the selected columns.
+    `new_column_name` must be a `Symbol` or a string, a vector of `Symbol` or
+    string, or `AsTable`, and `fun` a function or a type. If `old_column` is a
+    `Symbol`, a string, or an integer then `fun` is applied to the corresponding
+    column vector. Otherwise `old_column` can be any column indexing syntax, in
+    which case `fun` will be passed the column vectors specified by `old_column`
+    as separate arguments. The only exception is when `old_column` is an
+    `AsTable` type wrapping a selector, in which case `fun` is passed a
+    `NamedTuple` containing the selected columns.
+
+    Column renaming and transformation operations can be passed wrapped in
+    vectors or matrices (this is useful when combined with broadcasting).
+
+    # Rules when `new_column_name` is a `Symbol` or a string or is missing
```

Comment:
```suggestion
    # Rules when `new_column_name` is a `Symbol` or a string or is absent
```

Can you check that this gives a readable result in the REPL/manual given that it's a long line?

Reply:
It is OK (the text is 70 character long, so it would not look nice only for narrower terminals, but I think it is better to leave a longer description)